### PR TITLE
ncm-metaconfig: add mlocate

### PR DESCRIPTION
**Pull request by wdpypere**
Add metaconfig files for mlocate/slocate/updatedb.

### CODE REVIEW

File: ncm-metaconfig/src/main/metaconfig/mlocate/pan/schema.pan [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+include 'pan/types';
+
+@documentation{
+Override the default config file
+}
+type config_updatedb = {
+    @{A list of file system types which should not be scanned by updatedb.}
+    'prunefs' ? string[]
+    @{A list of directory names (without paths) which should not be scanned by updatedb.}
+    'prunenames' ? string[]
```
Comment (stdweird) [inline ncm-metaconfig/src/main/metaconfig/mlocate/pan/schema.pan]: there's a type for absolute paths

File: ncm-metaconfig/src/main/metaconfig/mlocate/pan/schema.pan [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+    @{A list of path names of directories which should not be scanned by updatedb.}
+    'prunepaths' ? string[]
```
Comment (stdweird) [inline ncm-metaconfig/src/main/metaconfig/mlocate/pan/schema.pan]: same remark, use type for absolute paths

File: ncm-metaconfig/src/main/metaconfig/mlocate/pan/schema.pan (hunk 1 above)
Comment (stdweird) [inline ncm-metaconfig/src/main/metaconfig/mlocate/pan/schema.pan]: hmm, `without paths`. nvm

**Comment by wdpypere** [comment]
@stdweird remarks fixed

**Pull request merged**